### ISSUE TITLE
Database can be shared?

### ISSUE THREAD

**Issue by yhyu13** [context-labs/autodoc#35]
Maybe a stupid question. So all content generated by autodoc and all contents autodoc required for querying is stored under the `.autodoc` folder, right? No other hidden local cache, right? As long as my server update this folder to my remote repo, I am able to share indexed database so that other developers don't have to manually index again?

Thanks!

**Comment by yhyu13** [comment]
A follow up question.

How does autodoc handle resursive `.autodoc` folder? For instance:

```
.autodoc
UE5/
UE5_game/
Fristparty/prioritory_plugin/.autodoc
```

Where UE5's auto doc is generated without Fristparty/prioritory_plugin being cloned. and Fristparty/prioritory_plugin's autodo is generated on its own.

**Comment by samheutmaker** [comment]
Yes your database can be shared. Other users who want to query your index will need to clone the repo and install autodoc globally via NPM. 

Recursive indexes are not supported at this time.
